### PR TITLE
[CALCITE-3938] Support LogicalCalc in RelShuttle

### PR DESCRIPTION
**Pull request by xy2953396112**
https://issues.apache.org/jira/browse/CALCITE-3938

### CODE REVIEW

File: core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java [hunk 1]
```diff
@@ -2106,6 +2111,27 @@ public final Sql sql(String sql) {
     assertTrue(filterCollation.satisfies(sortCollation));
   }
 
+  @Test void testRelShuttleForLogicalCalc() {
+    final String sql = "select ename from emp";
+    final RelNode rel = tester.convertSqlToRel(sql).rel;
+    final HepProgramBuilder programBuilder = HepProgram.builder();
+    programBuilder.addRuleInstance(ProjectToCalcRule.INSTANCE);
+    HepPlanner planner = new HepPlanner(programBuilder.build());
```
Comment (jinxing64) [inline core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java:2119]: final ？

**Comment by jinxing64** [comment]
How about rename the title as "Support LogicalCalc in RelShuttle" just as CALCITE-3607 ?

**Comment by xy2953396112** [comment]
> How about rename the title as "Support LogicalCalc in RelShuttle" just as [CALCITE-3607](https://issues.apache.org/jira/browse/CALCITE-3607) ?

ok, thanks for review.

**Comment by jinxing64** [comment]
Would you please trigger the test again ?

**Pull request merged**